samples/bluetooth/hci_rpmsg/nrf5340_cpunet_iso_receive.conf

```diff
@@ -22,22 +22,24 @@ CONFIG_BT_OBSERVER=y
 CONFIG_BT_CENTRAL=n
 CONFIG_BT_EXT_ADV=y
 # CONFIG_BT_PER_ADV=y
-# CONFIG_BT_ISO_BROADCASTER=y
 CONFIG_BT_PER_ADV_SYNC=y
+# CONFIG_BT_ISO_BROADCASTER=y
 CONFIG_BT_ISO_SYNC_RECEIVER=y
+CONFIG_BT_ISO_MAX_CHAN=2
 
 # ISO Broadcast Controller
-# CONFIG_BT_CTLR_ADV_EXT=y
 # CONFIG_BT_CTLR_ADV_PERIODIC=y
 # CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=191
 # CONFIG_BT_CTLR_ADV_ISO=y
 # CONFIG_BT_CTLR_ADV_ISO_PDU_LEN_MAX=251
-# CONFIG_BT_CTLR_ISO_TX_BUFFERS=3
+# CONFIG_BT_CTLR_ISO_TX_BUFFERS=16
 # CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE=251
+# CONFIG_BT_CTLR_ADV_ISO_STREAM_MAX=2
 
 # ISO Receive Controller
-CONFIG_BT_CTLR_ADV_EXT=y
 CONFIG_BT_CTLR_SYNC_PERIODIC=y
-CONFIG_BT_CTLR_SCAN_DATA_LEN_MAX=251
+CONFIG_BT_CTLR_SCAN_DATA_LEN_MAX=191
 CONFIG_BT_CTLR_SYNC_ISO=y
 CONFIG_BT_CTLR_SYNC_ISO_PDU_LEN_MAX=251
+CONFIG_BT_CTLR_SYNC_ISO_STREAM_MAX=2
```

Comment:
Do we need to increase `CONFIG_BT_CTLR_ISO_RX_BUFFERS`?

Reply:
No, Rx does not need buffering across ISO events, host shall in theory be able to receive every ISO data otherwise it would not meet the required real time playback.

Reply:
There is no required real time playback for ISO (arguably that may exist for audio, but in that case we also have the presentation delay).